### PR TITLE
fix #77644 lightnovelpub.com

### LINKED ISSUE #77644: www.lightnovelpub.com
### Issue URL (Ads)
 [https://www.lightnovelpub.com/novel/trash-count-wn-27022046/chapter-260](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.lightnovelpub.com%2Fnovel%2Ftrash-count-wn-27022046%2Fchapter-260)
### Comment
Username: @USFHunter
> There's this blank space ad leftover, but if i block it and used the " Block similar" option the ad-block gets detected. 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://www.lightnovelpub.com/novel/trash-count-wn-27022046/chapter-261
> 
> Custom userscripts:
> https://cdn.adguard.com/public/Userscripts/Beta/AdguardPopupBlocker/2.5/popupblocker.user.js
> https://raw.githubusercontent.com/reek/anti-adblock-killer/master/anti-adblock-killer.user.js
> https://cdn.adguard.com/public/Userscripts/AdguardAssistant/4.2/assistant.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v6.4.1544.4363
Browser: | Firefox
AdGuard driver: | WFP
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>AdGuard Base<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> EasyList China, <br/>EasyList Czech and Slovak, <br/>EasyList Italy, <br/>Liste AR, <br/>Icelandic ABP List, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>AdGuard Chinese, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese
Custom filter subscriptions: | https://raw.github.com/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt,<br>https://raw.githubusercontent.com/bogachenko/fuckfuckadblock/master/fuckfuckadblock.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#77644

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -568,6 +568,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77644
+lightnovelpub.com#%#//scriptlet("prevent-addEventListener", "DOMContentLoaded", "offsetHeight")
+!+ PLATFORM(ios, ext_android_cb, ext_safari)
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/antiadblock.txt:573]: scriptlets work in Safari
`!+ PLATFORM(ios, ext_android_cb)`

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:573]: How?

**Comment by Sergey-Lyapin** [comment]
@wout-codes
https://github.com/AdguardTeam/AdGuardForSafari/releases/tag/v1.4.1

I've made some changes =) https://github.com/AdguardTeam/AdguardFilters/commit/7c4e080fff2f78bf2f14963e9cc6e995c415bf53

**Comment by USFHunTer** [comment]
@Sergey-Lyapin I'm using firefox not safari so the fix didn't work for me, please update.

**Comment by wout-codes** [comment]
> @Sergey-Lyapin I'm using firefox not safari so the fix didn't work for me, please update.

Not reproduced. Try to update your filters, clear cache and cookies, then try.

**Comment by USFHunTer** [comment]
@wout-codes same result 
https://www.lightnovelpub.com/novel/the-beginning-after-the-end-novel-27022050/chapter-1
Annotation 2021-03-16 153915

**Comment by wout-codes** [comment]
Still not reproduced. I am not sure what's wrong on your end.
<details>

image
</details>

**Comment by USFHunTer** [comment]
it didn't work when I used the windows adguard to remove it, it just keeps coming back. But when i used the old browser Add-on "Adguard AdBlocker" to remove it worked i don't know why's that

**Comment by wout-codes** [comment]
I use AdGuard for Windows and it works on my end perfectly.

**Comment by USFHunTer** [comment]
I already have the latest stable release for windows 7.5.3

**Comment by wout-codes** [comment]
In the report it says that you're using `AdGuard for Windows v6.4.1544.4363`. I still don't get any leftovers or anti-adblock.

**Comment by USFHunTer** [comment]
yeah, I did, but I updated yesterday. it's rather after I've updated that blocking the AD didn't seem to do anything anymore, it's like the user filter is not working.

**Comment by Sergey-Lyapin** [comment]
I don't see leftover or antiadblock neither (with AG for Mac and Chrome extension), I'll check with windows app tonight.

**Comment by Sergey-Lyapin** [comment]
@USFHunTer
Please try to restart AdGuard and then restart browser and check this site in the incognito window.

**Pull request merged**